### ISSUE TITLE
Plug-in registered as [[pluginname]]

### ISSUE THREAD

**Issue by ratnavis** [mudrd8mz/moodle-editor_marklar#23]
I've installed the latest Marklar 1.0.0-beta.2 on a Moodle 3.9 on PHP 7.4.33. I kicked the install script all right. But under Plugins > Editors I see it registered as [[pluginname]]. Something is not good.

![image](https://user-images.githubusercontent.com/70723380/209683665-7d1b51ed-bca8-4ccb-b771-30d1369bf247.png)

Full version.php:
$plugin->component = 'editor_marklar';
$plugin->release = '1.0.0-beta.2';
$plugin->maturity = MATURITY_BETA;
$plugin->version = 2021060900;
$plugin->requires = 2019111800;
$plugin->supported = [39, 311];

**Comment by mudrd8mz** [comment]
> I kicked the install script all right

How exactly did you run the installation?

This may happen when the strings cache does not invalidate on the new plugin installation. But the install process should invalidate those caches. Can you try to clear the strings cache manually?

**Comment by ratnavis** [comment]
Installation went something like (not the real one, from memory):
$ cd MOODLE/lib/editor
$ $ git clone https://github.com/mudrd8mz/moodle-editor_marklar marklar

Visited the notifications page as 'admin', which then detected the new code and ran the install script. Didn't notice anything odd there. But the users are not shown Marklar in the Editor preferences > Text editor. As I wrote earlier the Plugins > Editors has that odd string. 

I ran purge all caches on the admin GUI. No change.
